scripts/variantstore/wdl/GvsCreateFilterSet.gvs_tieout_acmg_v2.inputs.json

```diff
@@ -1,41 +1,6 @@
-  "GvsCreateFilterSet.reference_dict": "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.dict",
-
-  "GvsCreateFilterSet.snp_recalibration_annotation_values": ["AS_QD", "AS_MQRankSum", "AS_ReadPosRankSum", "AS_FS", "AS_MQ", "AS_SOR"],
-  "GvsCreateFilterSet.snp_recalibration_tranche_values": ["100.0", "99.95", "99.9", "99.8", "99.6", "99.5", "99.4", "99.3", "99.0", "98.0", "97.0", "90.0" ],
-
-  "GvsCreateFilterSet.indel_recalibration_annotation_values": ["AS_FS", "AS_ReadPosRankSum", "AS_MQRankSum", "AS_QD", "AS_SOR"],
-  "GvsCreateFilterSet.indel_recalibration_tranche_values": ["100.0", "99.95", "99.9", "99.5", "99.0", "97.0", "96.0", "95.0", "94.0", "93.5", "93.0", "92.0", "91.0", "90.0"],
-
-  "GvsCreateFilterSet.axiomPoly_resource_vcf": "gs://gcp-public-data--broad-references/hg38/v0/Axiom_Exome_Plus.genotypes.all_populations.poly.hg38.vcf.gz",
-  "GvsCreateFilterSet.axiomPoly_resource_vcf_index": "gs://gcp-public-data--broad-references/hg38/v0/Axiom_Exome_Plus.genotypes.all_populations.poly.hg38.vcf.gz.tbi",
-  "GvsCreateFilterSet.dbsnp_vcf": "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.dbsnp138.vcf",
-  "GvsCreateFilterSet.dbsnp_vcf_index": "gs://gcp-public-data--broad-references/hg38/v0/Homo_sapiens_assembly38.dbsnp138.vcf.idx",
-  "GvsCreateFilterSet.hapmap_resource_vcf": "gs://gcp-public-data--broad-references/hg38/v0/hapmap_3.3.hg38.vcf.gz",
-  "GvsCreateFilterSet.hapmap_resource_vcf_index": "gs://gcp-public-data--broad-references/hg38/v0/hapmap_3.3.hg38.vcf.gz.tbi",
-  "GvsCreateFilterSet.mills_resource_vcf": "gs://gcp-public-data--broad-references/hg38/v0/Mills_and_1000G_gold_standard.indels.hg38.vcf.gz",
-  "GvsCreateFilterSet.mills_resource_vcf_index": "gs://gcp-public-data--broad-references/hg38/v0/Mills_and_1000G_gold_standard.indels.hg38.vcf.gz.tbi",
-  "GvsCreateFilterSet.omni_resource_vcf": "gs://gcp-public-data--broad-references/hg38/v0/1000G_omni2.5.hg38.vcf.gz",
-  "GvsCreateFilterSet.omni_resource_vcf_index": "gs://gcp-public-data--broad-references/hg38/v0/1000G_omni2.5.hg38.vcf.gz.tbi",
-  "GvsCreateFilterSet.one_thousand_genomes_resource_vcf": "gs://gcp-public-data--broad-references/hg38/v0/1000G_phase1.snps.high_confidence.hg38.vcf.gz",
-  "GvsCreateFilterSet.one_thousand_genomes_resource_vcf_index": "gs://gcp-public-data--broad-references/hg38/v0/1000G_phase1.snps.high_confidence.hg38.vcf.gz.tbi",
-
-  "GvsCreateFilterSet.small_disk": 100,
-  "GvsCreateFilterSet.medium_disk": 200,
-  "GvsCreateFilterSet.large_disk": 1000,
-  "GvsCreateFilterSet.huge_disk": 2000,
-
-  "GvsCreateFilterSet.gatk_override": "gs://broad-dsp-spec-ops/kcibul/gatk-package-4.2.0.0-312-g4fbcc6f-SNAPSHOT-local.jar",
-
-  "GvsCreateFilterSet.data_project": "spec-ops-aou",
-  "GvsCreateFilterSet.default_dataset": "gvs_tieout_acmg_v2",
-
+  "GvsCreateFilterSet.dataset_name": "gvs_tieout_acmg_v2",
```

Comment:
it occurs to me that this dataset is quite out of data and we may want to re-load the data if we are going to keep the example jsons around---why are we keeping the acmg json around?

Reply:
happy to delete it 👍🏻